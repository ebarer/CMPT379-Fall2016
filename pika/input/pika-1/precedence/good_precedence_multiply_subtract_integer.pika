exec {

  print 27 * 36 - 47, _n_.
  print 42 * 2 - 64, _n_.
  print 81 * 66 - 51, _n_.
  print 47 * 82 - 27, _n_.
  print 22 * 42 - 26, _n_.
  print 34 * 56 - 80, _n_.
  print 36 * 17 - 89, _n_.
  print 93 * 67 - 87, _n_.
  print 49 * 83 - 93, _n_.
  print 51 * 47 - 86, _n_.
  print 57 * 45 - 77, _n_.
  print 19 * 37 - 56, _n_.

  print 11 - 64 * 51, _n_.
  print 62 - 58 * 35, _n_.
  print 47 - 55 * 36, _n_.
  print 79 - 62 * 37, _n_.
  print 36 - 13 * 95, _n_.
  print 96 - 30 * 73, _n_.
  print 54 - 90 * 33, _n_.
  print 9 - 74 * 85, _n_.
  print 26 - 94 * 77, _n_.
  print 76 - 83 * 43, _n_.
  print 61 - 78 * 73, _n_.
  print 15 - 5 * 60, _n_.

  print 51 * 77 - 94 * 24, _n_.
  print 86 * 23 - 33 * 78, _n_.
  print 47 * 36 - 62 * 42, _n_.
  print 70 * 96 - 60 * 17, _n_.
  print 74 * 37 - 65 * 1, _n_.
  print 60 * 87 - 71 * 64, _n_.
  print 24 * 8 - 35 * 23, _n_.
  print 53 * 20 - 67 * 11, _n_.
  print 78 * 41 - 13 * 79, _n_.
  print 49 * 62 - 45 * 11, _n_.
  print 44 * 80 - 3 * 90, _n_.
  print 49 * 21 - 97 * 62, _n_.

  print 89 - 87 * 83 - 46, _n_.
  print 96 - 72 * 48 - 94, _n_.
  print 10 - 84 * 96 - 72, _n_.
  print 34 - 38 * 48 - 1, _n_.
  print 59 - 73 * 94 - 88, _n_.
  print 45 - 35 * 10 - 53, _n_.
  print 49 - 89 * 37 - 82, _n_.
  print 36 - 33 * 47 - 10, _n_.
  print 36 - 90 * 79 - 80, _n_.
  print 95 - 22 * 90 - 20, _n_.
  print 71 - 58 * 61 - 97, _n_.
  print 60 - 13 * 41 - 59, _n_.

}

