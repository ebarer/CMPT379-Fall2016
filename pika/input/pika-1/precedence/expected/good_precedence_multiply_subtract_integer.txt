925
20
5295
3827
898
1824
523
6144
3974
2311
2488
647
-3253
-1968
-1933
-2215
-1199
-2094
-2916
-6281
-7212
-3493
-5633
-285
1671
-596
-912
5700
2673
676
-613
323
2171
2543
3250
-4985
-7178
-3454
-8126
-1791
-6891
-358
-3326
-1525
-7154
-1905
-3564
-532
